package.json

```diff
@@ -69,7 +69,7 @@
     },
     "dependencies": {
         "@0x/assert": "^3.0.4",
-        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.4.0-07acc9529",
+        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.4.0-eb6b32b6e",
```

Comment:
⚠️ This should be updated w/ the merge commit once https://github.com/0xProject/0x-monorepo/pull/2562 is merged, correct? ⚠️